frontend/src/views/userDetail.js

```diff
@@ -53,12 +53,12 @@ export const UserDetail = ({ withHeader = true }) => {
   }, [navigate, token]);
 
   useEffect(() => {
-    if (token && username) {
-      fetchExternalJSONAPI(`${USER_STATS_API_URL}${username}`)
+    if (userDetails.id) {
+      fetchExternalJSONAPI(`${OHSOME_STATS_BASE_URL}/HotTMUser?userId=${userDetails.id}`, true)
```

Comment:
the api endpoint path has changed. It should be `/hot-tm-user`

https://stats.now.ohsome.org/api/swagger-ui/index.html#/access-restricted-user-controller/statsHotTMUser